File: android2_lesson01/src/java/java/com/example/ama/android2_lesson01/widget/ListNoteService.java
package com.example.ama.android2_lesson01.widget;

import android.content.Intent;
import android.widget.RemoteViewsService;

public class ListNoteService extends RemoteViewsService {

    @Override
    public RemoteViewsFactory onGetViewFactory(Intent intent) {
        return new ListNotesRemoteViewsFactory(getApplicationContext());
    }
}

File: android2_lesson01/src/kotlin/java/com/example/ama/android2_lesson01/widget/ListNoteService.kt
package com.example.ama.android2_lesson01.widget

import android.content.Intent
import android.widget.RemoteViewsService

class ListNoteService : RemoteViewsService() {
    override fun onGetViewFactory(intent: Intent?): RemoteViewsFactory =
            ListNoteRemoteViewsFactory(applicationContext)
}